rpcs3/main.cpp

```diff
@@ -105,20 +105,36 @@ const char* arg_styles     = "styles";
 const char* arg_style      = "style";
 const char* arg_stylesheet = "stylesheet";
 
-bool find_arg(std::string arg, int& argc, char* argv[])
+int find_arg(std::string arg, int& argc, char* argv[])
 {
 	arg = "--" + arg;
 	for (int i = 1; i < argc; ++i)
-		if (!strcmp(arg.c_str(), argv[i]))
-			return true;
-	return false;
+		if (!strncmp(arg.c_str(), argv[i], arg.size()))
```

Comment:
```suggestion
		if (arg == argv[i])
```

This is prone for false positives - `arg.size()` returns size without a null terminator, co cases where `arg = --arg` and `argv[i] == --argument` will return 0 as that **part** of the string is equal.

Since `arg` is a `std::string` (consider `std::string_view` maybe?), you can just check for equality.